### PR TITLE
Improve IPMA configuration documentation

### PR DESCRIPTION
**Pull request by dgomes**
## Proposed change

Help users understand the impact of their choice during configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ipma.markdown [hunk 1]
```diff
@@ -18,3 +18,7 @@ ha_integration_type: integration
 The `ipma` weather platform uses the [Instituto Português do Mar e Atmosfera](https://www.ipma.pt/) as a source for current and forecast meteorological data.
 
 {% include integrations/config_flow.md %}
+
+## Configuration
```
Comment (frenck) [inline source/_integrations/ipma.markdown]: This caused a duplicate header. I suggest to just remove it.

```suggestion
```

**Review by frenck — DISMISSED** [review]

**Review by klaasnicolaas — DISMISSED** [review]
✅ I think we can approve this one now

**Comment by klaasnicolaas** [comment]
@dgomes Could you merge the latest changes of `current` into your branch?

**Comment by dgomes** [comment]
done

**Comment by klaasnicolaas** [comment]
Just tried what Github offers because apparently it wasn't up to date yet and otherwise I can't merge 😅

**Review by klaasnicolaas — APPROVED** [review]

**Comment by dgomes** [comment]
Tks!

**Pull request merged**